polls/templates/polls/auth_token.html

```diff
@@ -30,12 +32,54 @@ <h3>API Token</h3>
 headers = {
     "Authorization": "Token {{ token.key }}"
     }
-url = "{{ admin_api_url }}"
+url = "{{ admin_api }}"
```

Comment:
Previous variable name: admin_api_url sounds better and clearer than "admin_api" as the former doest really have a great explanation for what it does.